pkg/migration/sqlmigrations/migrations_test.go

```diff
@@ -441,12 +447,17 @@ func TestCreateSystemTable(t *testing.T) {
 	settingsDescVal := sqlbase.WrapDescriptor(&settingsDesc)
 
 	// Start up a test server without running the system.jobs migration.
-	backwardCompatibleMigrations = []migrationDescriptor{
-		{
-			name:   "create system.settings table",
-			workFn: createSettingsTable,
-		},
+	// remove the view update migration so we can test its effects.
+	newMigrations := make([]migrationDescriptor, 0, len(backwardCompatibleMigrations))
+	for _, m := range backwardCompatibleMigrations {
+		if strings.HasPrefix(m.name, "create system.jobs") {
+			continue
+		}
+		newMigrations = append(newMigrations, m)
```

Comment:
Okay, thanks for the explanation. I fixed the test and added your explanation in a comment.